Documentation: Introduction to CVAT (#7229)
opencv/cvat: site/content/en/docs/getting_started/overview.md

  **available for free**. You can also choose a [**subscription**](https://www.cvat.ai/pricing/cloud)
  for **unlimited data**, **organizational tools**, **auto-annotations**, and [**more**](https://www.cvat.ai/post/cvat-ai-pricing-plans-choosing-the-right-plan-for-your-needs).

- **Self-hosted CVAT** (forever Free of charge): Follow the [**Self-hosted Installation Guide**](/docs/administration/basics/installation/)
  for setup. We provide [**Enterprise-level support**](https://www.cvat.ai/pricing/on-prem)
  for this version, **including premium features** like **SSO**, **LDAP**, advanced integrations with
  [**Roboflow and HuggingFace**](https://www.cvat.ai/post/integrating-hugging-face-and-roboflow-models),
  and **advanced analytics**. We also offer **professional training** and **24-hour SLA support**.

See:

- [Tools and formats](#tools-and-formats)
  - [Supported formats](#supported-formats)
  - [Annotation tools](#annotation-tools)
- [Automated labeling](#automated-labeling)
- [Useful links](#useful-links)
- [Integrations](#integrations)
- [License Information](#license-information)
- [Get in touch](#get-in-touch)

## Tools and formats

> CVAT stands as a comprehensive

Reviewer: Could you please fix formatings? I see a couple of words on each line. Why?

Author: @nmanovic I do not see why the formatting must be fixed. It looks ok for me. You see a couple of words on each line, because I cut them this way for the linter not to throw an error for the lines to be too long. Docsy does not see it as "cut"  anyway and the final version looks fine. 
![image](https://github.com/opencv/cvat/assets/39969264/cc18a3c6-c652-46d3-89ac-5da154f17b11)

Reviewer: Why don't one word per line in this way? Please make it appropriate. The recommended size of a line is around 80 characters (I believe our linter complains when it is 120). It is weird when the first line is only 30 without any reason. It is just UNREADABLE.

tool for image and video annotation,
essential for various computer vision tasks.
It emphasizes user-friendliness,
adaptability, and compatibility with a range of formats and tools.

### Supported formats

CVAT's supports the following formats:

- **For 3D**: `.pcd`, `.bin`

> - **For image**: everything supported by the Python Pillow library,

Reviewer: Probably it is good to add a link on formats supported by PIL: https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html

  including formats like `JPEG`, `PNG`, `BMP`, `GIG`, `PPM`and `TIFF`.
- **For video**: all formats, supported by ffmpeg, including `MP4`, `AVI`, and `MOV`.

For annotation export and import formats, see
[**Export annotations and data from CVAT**](/docs/manual/advanced/formats/)

### Annotation tools

CVAT offers a wide array of annotation tools,
each catering to different aspects of image and video labeling:

> - **3D Object Annotation**: For annotating objects in three dimensions,

Reviewer: I would recommend to insert here just links
`[3D Object Annotation](relative path to documentation)`

I don't think that such "obvious" description makes sense.

Author: @nmanovic I've added links but would like to save the descriptions. Descriptions give small advice that users sometimes ask in Discord for example.  I will elaborate on them a bit more in some kind of next iteration, this page will need a couple of updates shortly (for example when I will finish updating all the tools docs).

  providing a comprehensive perspective.
- **Attribute Annotation Mode**: Allows adding specific
  attributes to annotations.
- **Annotation with Rectangles**: Ideal for straightforward
  object detection tasks.
- **Annotation with Polygons**: Perfect for detailed
  segmentation of irregular shapes.
- **Annotation with Polylines**: Useful for tracing
  elongated shapes or paths.
- **Annotation with Ellipses**: Suitable for objects
  with elliptical shapes.
- **Annotation with Cuboids**: For 3D representation
  of objects in two dimensions.
- **Annotation with Skeletons**: Useful in pose estimation and
  tracking of articulated objects.
- **Annotation with Brush Tool**: Allows freehand drawing for
  detailed and complex annotations.
- **Annotation with Tags**: Tagging images or video frames for
  classification or categorization.

These tools make CVAT a versatile platform for a range of annotation
needs, from basic labeling to complex, multidimensional
tasks in advanced computer vision projects.

## Automated labeling

> CVAT has an automated labeling features,

Reviewer: Again, strage formating. 80 chars in a line should work.

Author: @nmanovic same comment as above... if you mean just formatting here. Or it look different for you when you check it on localhost?

Reviewer: Please ask if you cannot interpret the guidelines. It is markdown. It should be readable. When you less than half of a page, it is not BKMs for sure.

len("CVAT has an automated labeling features,") is 40 characters!

<!--lint enable maximum-line-length-->

## Integrations

CVAT is a global tool, trusted and utilized by teams worldwide.
Below is a list of key companies that contribute significantly to our
product support or are an integral part of our ecosystem.

> **Note:** If you're using CVAT, we'd love to
> hear from you at [contact@cvat.ai](mailto:contact+github@cvat.ai).

- [**Human Protocol**](https://hmt.ai): This platform incorporates CVAT
  for augmenting its annotation services within the Human Protocol framework.
- [**FiftyOne**](https://fiftyone.ai): As an open-source tool for dataset
  management and model analysis in computer vision,
  FiftyOne is [closely integrated](https://voxel51.com/docs/fiftyone/integrations/cvat.html)
  with CVAT, enhancing annotation capabilities and label refinement.
- [**Toloka**](https://toloka.ai): Toloka leverages CVAT for its crowdsourced data labeling
  and annotation services, enriching its

>   diverse task handling capabilities. For mre information, see [**Integration with Toloka**](/docs/integration/toloka/)

Reviewer: Probably it is good to add information about Hugging Face and Roboflow, but the primary question how to combine CVAT.ai Cloud and GitHub documentation in one place.

## License Information

> This code is distributed under the [**MIT License**](https://opensource.org/licenses/MIT),

Reviewer: Again, the license is only for code inside CVAt public GitHub repo. For the enterprise, we have a commercial license.

a permissive free software license.

Additionally, this software incorporates LGPL-licensed components from
the [**FFmpeg**](https://www.ffmpeg.org) project.
For more details, visit [**FFMPEG Legal**](https://www.ffmpeg.org/legal.html).
Users must independently verify whether their use of FFmpeg necessitates
additional licenses.
CVAT.ai Corporation does not provide these licenses
and is not liable for any related licensing fees incurred through the use of FFmpeg.

For terms of use and confidential information handling,
see [**Terms of Use**](https://www.cvat.ai/terms-of-use).

## Get in touch

> **Note:** For **commercial support
> inquiries**, please email us directly at [**contact@cvat.ai**](mailto:contact+github@cvat.ai).

> - [**Gitter community**](https://gitter.im/opencv-cvat/public): For questions about using CVAT.

Reviewer: Let's exclude Gitter. We are going to develop and grow only Discord.

  This platform is actively monitored by our core team and the broader community,
  ensuring quick responses to questions. You can also find answers to frequently asked questions here.

- [**Discord channel**](https://discord.gg/S6sRHhuQ7K). For broader
  discussions and questions, head over to our. This is a space for all things related to CVAT.
- [**LinkedIn**](https://www.linkedin.com/company/cvat-ai/): For company updates and employment opportunities.

- [**YouTube channel**](https://www.youtube.com/@cvat-ai): For tutorials and screencasts about CVAT tools.

> - [**GitHub issues**](https://github.com/cvat-ai/cvat/issues): To report bugs.

Reviewer: We also have Customer support channel for CVAT.ai cloud paid users:https://youtrack.cvat.ai/form/447d9c98-ab4b-466e-bf9d-004f01b22f73

Probably we want to specify it as well. Again, need to somehow separate it from the open-source version. Please think how to do that.

---
title: 'CVAT Overview'
linkTitle: 'Introduction to CVAT'
weight: 1

> description: 'The open-source tool for image and video annotation and data labeling'

Reviewer: ```suggestion
description: 'The open-source tool for image and video annotation'
```

> description: 'CVAT: The open-source tool for image and video annotation and data labeling'

Reviewer: ```suggestion
description: 'The open-source tool for image and video annotation and data labeling'
```

---

> Machine learning systems often struggle due to

Reviewer: Please fix formatting across the whole document. Keep the line length is ~80 characters. You can use some tools to fix that.

improving a model can be tough and inefficient.

[**CVAT.ai**](https://www.cvat.ai/) **is a** versatile **tool** for
**annotating images and videos**, serving the computer
vision community worldwide.

Our goal is to help developers, businesses,
and organizations globally by using a Data-centric AI approach.

> CVAT offers two versions:

Reviewer: ```suggestion
CVAT offers three versions:
```

CVAT offers two versions:

- [**CVAT Cloud**](https://app.cvat.ai/): Start online with CVAT,
  **available for free**. You can also choose a [**subscription**](https://www.cvat.ai/pricing/cloud)

>   for **unlimited data**, **organizational tools**, **auto-annotations**, and [**more**](https://www.cvat.ai/post/cvat-ai-pricing-plans-choosing-the-right-plan-for-your-needs).

Reviewer: What does "organization tools" mean? Could you please rephrase. I'm not sure that we have such term in our documentation.

> - **Self-hosted CVAT** (forever Free of charge): Follow the [**Self-hosted Installation Guide**](/docs/administration/basics/installation/)

Reviewer: ```suggestion
- **Self-hosted CVAT Community Edition**: Follow the [**Self-hosted Installation Guide**](/docs/administration/basics/installation/)
```

>   for setup. We provide [**Enterprise-level support**](https://www.cvat.ai/pricing/on-prem)

Reviewer: I would distinguish between "Community edition" and "Enterprise edition". They are two separate products.

  for this version, **including premium features** like **SSO**, **LDAP**, advanced integrations with
  [**Roboflow and HuggingFace**](https://www.cvat.ai/post/integrating-hugging-face-and-roboflow-models),
  and **advanced analytics**. We also offer **professional training** and **24-hour SLA support**.

See:

- [Tools and formats](#tools-and-formats)
  - [Supported formats](#supported-formats)
  - [Annotation tools](#annotation-tools)
- [Automated labeling](#automated-labeling)
- [Useful links](#useful-links)
  - [Cloud](#cloud)
  - [Self-Hosted](#self-hosted)
- [Integrations](#integrations)
- [License Information](#license-information)
- [Get in touch](#get-in-touch)

## Tools and formats

CVAT stands as a comprehensive tool for image and video annotation,
essential for various computer vision tasks.

It emphasizes user-friendliness, adaptability, and compatibility with
a range of formats and tools.

### Supported formats

CVAT's supports the following formats:

- **For 3D**: `.pcd`, `.bin`
- **For image**: everything supported by the Python
  [**Pillow library**](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html),

>   including formats like `JPEG`, `PNG`, `BMP`, `GIG`, `PPM`and `TIFF`.

Reviewer: ```suggestion
  including formats like `JPEG`, `PNG`, `BMP`, `GIF`, `PPM`and `TIFF`.
```

For annotation export and import formats, see
[**Export annotations and data from CVAT**](/docs/manual/advanced/formats/)

### Annotation tools

> CVAT offers a wide array of annotation tools,

Reviewer: array or range?

| [**Annotation with Tags**](/docs/manual/advanced/annotation-with-tags/)                    | Useful for image and video classification tasks, like identifying scenes or themes in a dataset.               |

<!--lint enable maximum-line-length-->

These tools make CVAT a versatile platform for a range of annotation
needs, from basic labeling to complex, multidimensional
tasks in advanced computer vision projects.

## Automated labeling

CVAT has an automated labeling features,
enhancing the annotation process significantly,
potentially speeding it up by up to 10 times.

> **Note:**
> For more information,
> see [**OpenCV and AI Tools**](/docs/manual/advanced/ai-tools/)

Below is a detailed table of the supported algorithms and the platforms they operate on:

<!--lint disable maximum-line-length-->

| Algorithm Name                                                                                          | Category   | Framework  | CPU Support | GPU Support |
| ------------------------------------------------------------------------------------------------------- | ---------- | ---------- | ----------- | ----------- |

> | [Segment Anything](/serverless/pytorch/facebookresearch/sam/nuclio/)                                    | Interactor | PyTorch    | ✔️          | ✔️          |

Reviewer: All these links don't work by an obvious reason. Could you please check all other links in the article?

| [**CVAT Complete Workflow Guide for Organizations**](/docs/getting_started/workflow-org/) | This guide provides a comprehensive overview of using CVAT for collaboration in organizations.                                                                                                                                                                      |
| [**Subscription Management**](/docs/enterprise/subscription-managment/)                   | Learn how to [**choose a plan**](https://www.cvat.ai/post/cvat-ai-pricing-plans-choosing-the-right-plan-for-your-needs), subscribe, and manage your subscription effectively.                                                                                       |
| [**XML Annotation Format**](/docs/manual/advanced/xml_format/)                            | Detailed documentation on the XML format used for annotations in CVAT essential for understanding data structure and compatibility.                                                                                                                                 |

> | [**Frequently Asked Questions**](/cvat/docs/faq/)                                         | This section addresses common queries and provides helpful answers and insights about using CVAT.                                                                                                                                                                   |

Reviewer: The link doesn't work as well.

| [**Command Line Tool**](/docs/api_sdk/cli/)                                                    | This tool offers a straightforward command line interface for managing CVAT tasks. Currently featuring basic functionalities, it has the potential to develop into a more advanced administration tool for CVAT.           |
| [**XML Annotation Format**](/docs/manual/advanced/xml_format/)                                 | Detailed documentation on the XML format used for annotations in CVAT essential for understanding data structure and compatibility.                                                                                        |
| [**AWS Deployment Guide**](/docs/administration/basics/aws-deployment-guide/)                  | A step-by-step guide for deploying CVAT on Amazon Web Services, covering all necessary procedures and tips.                                                                                                                |

> | [**Frequently Asked Questions**](/cvat/docs/faq/)                                              | This section addresses common queries and provides helpful answers and insights about using CVAT.                                                                                                                          |

Reviewer: I see the link twice. I'm not sure it should be here. It is a common FAQ.

| [**Human Protocol**](https://hmt.ai)                                                                   | Cloud and Self-hosted | Incorporates CVAT to augment annotation services within the Human Protocol framework, enhancing its capabilities in data labeling.                                                                                                              |
| [**FiftyOne**](https://fiftyone.ai)                                                                    | Cloud and Self-hosted | An open-source tool for dataset management and model analysis in computer vision, FiftyOne is [closely integrated](https://voxel51.com/docs/fiftyone/integrations/cvat.html) with CVAT to enhance annotation capabilities and label refinement. |
| [**Toloka**](https://toloka.ai)                                                                        | Cloud                 | Utilizes CVAT for crowdsourced data labeling and annotation services, enriching Toloka's diverse task handling capabilities. For more information, see [**Integration with Toloka**](/docs/integration/toloka/).                                |

> | [**Hugging Face and Roboflow**](https://www.cvat.ai/post/integrating-hugging-face-and-roboflow-models) | Cloud                 | In CVAT Cloud, models from [**Hugging Face**](https://huggingface.co/) and [**Roboflow**](https://roboflow.com/) can be added to enhance computer vision tasks.                                                                                 |

Reviewer: I'm not sure why all previous links were to services, and this one to documentation?

Author: @nmanovic it is the only place where integration is described. There are links to the services in the description.

Author: UPD: nevermind my previous comment, I ll redo it the same way as Toloka

| [**MIT License**](https://opensource.org/licenses/MIT) | Self-hosted            | This code is distributed under the MIT License, a permissive free software license that allows for broad use, modification, and distribution.                                                                                                 |
| [**LGPL License (FFmpeg)**](https://www.ffmpeg.org)    | Cloud and Self-hosted  | Incorporates LGPL-licensed components from the FFmpeg project. Users should verify if their use of FFmpeg requires additional licenses. CVAT.ai Corporation does not provide these licenses and is not liable for any related licensing fees. |
| **Commercial License**                            | Self-hosted Enterprise | For commercial use of the Enterprise solution of CVAT, a separate commercial license is applicable. This is tailored for businesses and commercial entities.                                                                                  |

> | [**Terms of Use**](https://www.cvat.ai/terms-of-use)   | Cloud and Self-hosted  | Outlines the terms of use and confidential information handling for CVAT. Important for understanding the legal framework of using the platform.                                                                                              |

Reviewer: Need to add privacy policy into the list
